packages/ui/VirtualList/VirtualListBasic.js

```diff
@@ -577,6 +585,7 @@ class VirtualListBasic extends Component {
 		}
 
 		position.primaryPosition -= offset;
+		position.primaryPosition = clamp(0, maxPos, position.primaryPosition);
```

Comment:
Could you merge this two lines?

		position.primaryPosition = clamp(0, maxPos, position.primaryPosition - offset);

Reply:
👌